### PR TITLE
Test fix

### PR DESCRIPTION
**Pull request by shortChangeDev**
#### What changes does this PR contain?
Quick fix test for wtf Unix bug 32768 A N G E R Y B O Y S 
#### Notes for reviewers

@ubivius/gamedev

### CODE REVIEW

File: Assets/Scripts/TCPToolkit/TCPClient.cs [hunk 1]
```diff
@@ -216,7 +214,7 @@ private void ReceivingThread(object streamObj)
                 }
             }
 #if DEBUG_LOG
-            Debug.Log("State at client receiving thread exit : Active endpoint ? " + m_activeEndpoint.ToString() + ", Exit signal ?" + m_exitSignal + ", Buffer offset :" + bufferOffset);
+            Debug.Log("State at client receiving thread exit : Active endpoint ? " + m_activeEndpoint.ToString() + ", Exit signal ?" + m_exitSignal);
```
Comment (ysma500) [inline Assets/Scripts/TCPToolkit/TCPClient.cs:217]: Didn't say anything about this yesterday but concatenation of string with "+" normally is heavier to process than a good old String.Format("State at client receiving thread exit : Active endpoint ? {0}, Exit signal ? {1}", m_activeEndpoint.ToString(), m_exitSignal)

File: Assets/Scripts/TCPToolkit/TCPToolkit.cs [hunk 2]
```diff
@@ -63,9 +63,15 @@ public static Packet FirstPacketFromBytes(byte[] bytes)
                     }
 
                     int byteCount = packet.DataSize + TCP_HEADER_SIZE;
+                    
+                    if (bytes.Length < byteCount)
+                    {
+                        Debug.Log("Packet not ready to be built yet: " + bytes.Length + " vs " + byteCount);
```
Comment (ysma500) [inline Assets/Scripts/TCPToolkit/TCPToolkit.cs:69]: Same as the other, can we change to: Debug.Log(String.Format("Packet not ready to be built yet: {0} vs {1}", bytes.Length, byteCount)); ?

**Review by ysma500 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/TCPToolkit/TCPToolkit.cs (hunk 2 above)
Comment (ysma500) [reply Assets/Scripts/TCPToolkit/TCPToolkit.cs:69]: Apparently, I'm very wrong in c# as seen [here](https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java). Therefore, I'll shut up and approve this PR.

File: Assets/Scripts/TCPToolkit/TCPClient.cs (hunk 1 above)
Comment (ysma500) [reply Assets/Scripts/TCPToolkit/TCPClient.cs:217]: Apparently, I'm very wrong in c# as seen [here](https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java). Therefore, I'll shut up and approve this PR.

**Review by ysma500 — APPROVED** [review]
LGTM!

File: Assets/Scripts/TCPToolkit/TCPClient.cs [hunk 3]
```diff
@@ -32,7 +32,9 @@ public class TCPClient : MonoBehaviour
         private List<ITCPClientReceiver> m_receiversAwaitingUnsubscription;
         private bool m_iteratingTroughReceivers;
 
-        private const int DATA_BUFFER_SIZE = 1024*1024*4;
+
+        private const int DATA_BUFFER_SIZE = 1024 * 1024 * 4;
```
Comment (JerPaq) [inline Assets/Scripts/TCPToolkit/TCPClient.cs:36]: How dare you put space between number and operator

File: Assets/Scripts/TCPToolkit/TCPClient.cs [hunk 4]
```diff
@@ -32,7 +32,9 @@ public class TCPClient : MonoBehaviour
...
+        private const int MAX_BYTES_READ = 32768;
```
Comment (JerPaq) [inline Assets/Scripts/TCPToolkit/TCPClient.cs:37]: Pas très utile, mais peut-être une commentaire qui dit pourquoi le max est cette valeur.

File: Assets/Scripts/TCPToolkit/TCPServer.cs [hunk 5]
```diff
@@ -27,7 +27,8 @@ public class TCPServer : MonoBehaviour
         protected volatile bool m_exitSignal;
 
         private const int DATA_BUFFER_SIZE = 1024*1024*4;
-                
+        private const int MAX_BYTES_READ = 32768;
```
Comment (JerPaq) [inline Assets/Scripts/TCPToolkit/TCPServer.cs:30]: Même chose sur le nombre et le commentaire

**Review by JerPaq — APPROVED** [review]
AIDIWWWNMW

**Review by Daftdoritos — APPROVED** [review]
Lets go!

**Pull request merged**